tests/tests_suite_test.go

```diff
@@ -140,6 +151,19 @@ func (s *newSspStrategy) GetValidatorReplicas() int {
 	return templateValidatorReplicas
 }
 
+func (s *newSspStrategy) GetVersionLabel() string {
+	if s.ssp.Labels == nil {
```

Comment:
No need to check for `nil` here, because it is set to a non-nil value in `Init()`.